.github/workflows/ci.yml

```diff
@@ -0,0 +1,86 @@
+name: CI
+
+# On every pull request, but only on push to master
+on:
+  push:
+    branches:
+    - master
+    tags:
+    - '*'
+  pull_request:
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [2.7, 3.7, 3.8, 3.9]
```

Comment:
```suggestion
        python-version: [3.6, 3.7, 3.8, 3.9, 3.10]
```

Reply:
I didn't see that we didn't support 2.7 anymore, maybe it will be good to update setup.py